ouroboros-consensus-cardano-tools/ouroboros-consensus-cardano-tools.cabal

```diff
@@ -0,0 +1,186 @@
+
+flag asserts
+  description: Enable assertions
+  manual:      False
+  default:     False
+
+library
+  hs-source-dirs:      src
+
+  exposed-modules:     Cardano.Api.Any
+                     , Cardano.Api.Protocol.Types
+                     , Cardano.Node.Protocol.Types
+                     , Cardano.Node.Types
+                     , Cardano.Node.Protocol
+                     , Cardano.Tools.DBAnalyser.Analysis
+                     , Cardano.Tools.DBAnalyser.Block.Byron
+                     , Cardano.Tools.DBAnalyser.Block.Cardano
+                     , Cardano.Tools.DBAnalyser.Block.Shelley
+                     , Cardano.Tools.DBAnalyser.HasAnalysis
+                     , Cardano.Tools.DBAnalyser.Run
+                     , Cardano.Tools.DBAnalyser.Types
+                     , Cardano.Tools.DBSynthesizer.Forging
+                     , Cardano.Tools.DBSynthesizer.Orphans
+                     , Cardano.Tools.DBSynthesizer.Run
+                     , Cardano.Tools.DBSynthesizer.Types
+
+  build-depends:       base
+                     , aeson
+                     , base16-bytestring >= 1.0
+                     , bytestring
+                     , cborg
+                     , containers
+                     , contra-tracer
+                     , deepseq
+                     , directory
+                     , filepath
+                     , mtl
+                     , nothunks
+                     , serialise
+                     , strict-containers
+                     , text
+                     , transformers
+                     , transformers-except
+
+                     , cardano-binary
+                     , cardano-crypto
+                     , cardano-crypto-wrapper
+                     , cardano-crypto-class
+                     , cardano-ledger-alonzo
+                     , cardano-ledger-babbage
+                     , cardano-ledger-byron
+                     , cardano-ledger-core
+                     , cardano-ledger-shelley
+                     , cardano-protocol-tpraos
+                     , cardano-prelude
+
+                     , ouroboros-consensus
+                     , ouroboros-consensus-byron
+                     , ouroboros-consensus-cardano
+                     , ouroboros-consensus-protocol
+                     , ouroboros-consensus-shelley
+                     , ouroboros-network
+
+  other-modules:       Cardano.Api.Key
+                     , Cardano.Api.KeysByron
+                     , Cardano.Api.KeysPraos
+                     , Cardano.Api.KeysShelley
+                     , Cardano.Api.OperationalCertificate
+                     , Cardano.Api.SerialiseUsing
+                     , Cardano.Api.SerialiseTextEnvelope
+                     , Cardano.Node.Protocol.Alonzo
+                     , Cardano.Node.Protocol.Byron
+                     , Cardano.Node.Protocol.Cardano
+                     , Cardano.Node.Protocol.Shelley
```

Comment:
That's another question of negotiation, I guess.

I reckon that @nfrisby has already implemented the "node config"-based configuration for the analyser -- so maybe we need to join efforts there.

Reply:
The "node-config" based analyser code is already present in this PR. @nfrisby thankfully notified me when he merged the corresponding feature branch into `master`, so I could port the relevant changes.

A follow-up consideration is part of the draft issue `cardano-tools: code deduplication & future maintenance` in https://github.com/orgs/input-output-hk/projects/22/views/1: using "node-config" in the analyser has opened up room for removing redundant code for both tools.

The decision would be, since that does not change behaviour or API at all, if we want that to be in scope for this PR, or whether it's better suited for a future next step.